File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/ReferralPerformanceRating.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class ReferralPerformanceRating
    {
        public int ReferralPerformanceRatingId { get; set; }
        public ReferralRatingEnum ReferralRating { get; set; }
        public string Comment { get; set; }
        public bool IsDeleted { get; set; }
        public string ModifiedBy { get; set; }
        public System.DateTime ModifiedDate { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/referrals/referral-performance-rating.ts
import { ReferralRatingEnum } from "./referral-rating-enum";

export class ReferralPerformanceRating {
    referralPerformanceRatingId: number;
    referralRating: ReferralRatingEnum;
    comment: string;
    isDeleted: boolean;
    modifiedBy: string;
    modifiedDate: Date;
    createdBy: string;
    createdDate: Date;
}
